Firmware/ultralcd.cpp

```diff
@@ -2057,8 +2057,8 @@ static void lcd_support_menu()
 	{	// 22bytes total
 		int8_t status;                 // 1byte
 		bool is_flash_air;             // 1byte
-		uint8_t ip[4];                 // 4bytes
-		char ip_str[3*4+3+1];          // 16bytes
+		uint32_t ip;                   // 4bytes
+		char ip_str[IP4_STR_SIZE];     // 16bytes
```

Comment:
do you really need to store the formatted string? Is printing 4 numbers so slow?

Reply:
No. We store stuff in memory since that's how the stupid LCD function works. I could probably avoid using this, but it's menu_data so it's not taking any extra space